pkgs/tools/networking/keepalived/default.nix

```diff
@@ -1,33 +1,17 @@
-{ stdenv, fetchurl, openssl, net_snmp, libnl }:
```

Comment:
Why did you remove support libnl and net_snmp? Sounds generally useful to me. Though libnl should only be enabled if this is build on Linux.